src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss

```diff
@@ -2,95 +2,55 @@
 //@import "../PropertyPaneFilePicker.module.scss";
 
 /** TILES **/
-.filePickerFolderCardGrid {
-  overflow: hidden;
-  font-size: 0;
-
-  :global {
-    .ms-List-page {
-      overflow: hidden;
-      font-size: 0;
-    }
-
-    .ms-List-surface {
-      position: relative;
-    }
-  }
+.documentLibraryBrowserContainer {
+  padding: 0;
 }
 
 .filePickerFolderCardTile {
+  max-width: 176px;
```

Comment:
From what I tested, this property is not mandatory, as it's already calculated from the whole div.